nmtwizard/preprocess/consumer.py

```diff
@@ -137,10 +141,25 @@ def _log_info(file_info, file_summary, proc, units):
             summary = self._summary[proc]
             if summary:
                 all_files_summary = summary.pop(None)
-                _log_info("for the whole sample ", all_files_summary, proc, units)
+                total_sentence_number = sum(
+                    [v.get("linesampled") for v in self._sampler_summary.values()]
+                )
+                _log_info(
+                    "for the whole sample ",
+                    all_files_summary,
+                    proc,
+                    units,
+                    total_sentence_number,
+                )
                 sorted_by_base_name = sorted(summary.items(), key=lambda item: item[0])
                 for base_name, file_summary in sorted_by_base_name:
-                    _log_info(f"for the file '{base_name}' ", file_summary, proc, units)
+                    _log_info(
+                        f"for the file '{base_name}' ",
+                        file_summary,
+                        proc,
+                        units,
+                        self._sampler_summary.get(base_name, {}).get("linesampled"),
```

Comment:
Same comment regarding the "linesampled" field.

Reply:
I think "linesampled" field is always present, same goes for the file name. So I replaced get with a simple lookup.